src/demo-app/sidenav/sidenav-demo.html

```diff
@@ -15,7 +15,7 @@
     <input #myinput>
   </md-sidenav>
 
-  <md-sidenav #end align="end">
+  <md-sidenav #end side="end">
```

Comment:
That sounds good to me, I just didn't like `align`, sounded like it was aligning the sidenav contents or something